test/test_dht.cpp

```diff
@@ -1069,7 +1069,7 @@ TORRENT_TEST(dht)
 		printf("num_active_buckets: %d\n", tbl.num_active_buckets());
 		TEST_EQUAL(tbl.num_active_buckets(), 6);
    
-#if defined TORRENT_DHT_VERBOSE_LOGGING || defined TORRENT_DEBUG
```

Comment:
yes. I believe I did mean && there. I'd be happy if you'd just comment this call out for now. Ideally this data should be accessible programmatically and be loggable by the client.